a10_neutron_lbaas/v1/handler_vip.py

```diff
@@ -190,20 +195,26 @@ def s_persistence(self):
         return self.s_pers
 
     def create(self):
-        if self.sp is None:
-            return
+        # TODO(HUNTER) - Assign source data and function pointer then just call
+        # function pointer with the data you want.
+        if self.old_vip is not None:
+            vip_sp = self.old_vip.get("session_persistence")
+            if vip_sp is not None:
+                try:
+                    vip_sp_type = vip_sp.get("type")
+                    m = getattr(self.c.client.slb.template, self.sp_obj_dict[vip_sp_type])
+                    m.delete(self.old_vip.get("id"))
+                except Exception:
```

Comment:
This should be acos_errors.Exists - catching all exceptions and eating them here is bound to hide bugs.